### PR TITLE
Update the README for the Zashi iOS repository.

### PR DESCRIPTION
**Pull request by nuttycom**
This depends on [Electric-Coin-Company/zashi#1]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
+`good first issue` label. Please fork the repo and make a pull request for us
+to review.
 
-This a reference wallet for the following set of features:
-- z2z transactions w/ encrypted memos
-- reply-to formatted memos
-- z2t transactions
-- transparent receive-only
-- autoshielding on threshold from receive only t-address
+Secant Wallet uses [SwiftLint](https://github.com/realm/SwiftLint) and
```
Comment (pacu) [inline README.md:38]:
```suggestion
Zashi Wallet uses [SwiftLint](https://github.com/realm/SwiftLint) and 
```

File: README.md [hunk 2]
```diff
@@ -1,40 +1,50 @@
-# secant-ios-wallet
+# Zashi IOS Wallet
```
Comment (pacu) [inline README.md:1]:
```suggestion
# Zashi iOS Wallet
```

File: README.md [hunk 3]
```diff
@@ -1,40 +1,50 @@
...
-- Traffic analysis, like in other cryptocurrency wallets, can leak some privacy of the user.
-- The wallet requires a trust in the server to display accurate transaction information. 
+# Reporting an issue
+
+If you'd like to report a technical issue or feature request for the IOS
+Wallet, please file a GitHub issue [here](https://github.com/Electric-Coin-Company/zashi-ios/issues/new/choose).
 
-See the [Wallet App Threat Model](https://zcash.readthedocs.io/en/latest/rtd_pages/wallet_threat_model.html)
+For feature requests and issues related to the Zashi user interface that are
+not IOS-specific, please file a GitHub issue [here](https://github.com/Electric-Coin-Company/zashi/issues/new/choose).
```
Comment (pacu) [inline README.md:20]:
```suggestion
not iOS-specific, please file a GitHub issue [here](https://github.com/Electric-Coin-Company/zashi/issues/new/choose).
```

File: README.md [hunk 4]
```diff
@@ -1,40 +1,50 @@
-# secant-ios-wallet
+# Zashi iOS Wallet
 
-This wallet is a Dogfooding effort towards Zcash Halo Arc / NU5 efforts.
+This is the official home of the Zashi Zcash wallet for Wallet, a no-frills
+Zcash mobile wallet leveraging the [Zcash Swift SDK](https://github.com/zcash/ZcashLightClientKit).
```
Comment (LukasKorba) [inline README.md:4]: Ideally the link is already the one to the ECC SDK repo.

**Review by LukasKorba — APPROVED** [review]

**Pull request merged**